services/job_runner.go

```diff
@@ -368,6 +370,15 @@ func wrapExecuteRunAtBlockError(run models.JobRun, err error) error {
 	return nil
 }
 
+func logNodeBalances(store *store.Store) {
+	kv, err := presenters.ShowEthBalance(store)
```

Comment:
I thought about adding a flag that can disable the log for this reason, but then it's just adding more config variables.

This could be something I monitor externally to the node and alert for, but then this could be a potential pain point for other operators who don't top up their nodes ETH balance. I feel there needs to be some sort of mechanism that can notify operators when they're running low.

Reply:
I'd say if you can ask the logger if debug mode is on, this might be acceptable, e.g:

```go
if logger.Level() == DEBUG {
  logNodeBalances(store)
}
```

(this is pseudocode)